package.json

```diff
@@ -17,7 +17,7 @@
     "@bcrs-shared-components/approval-type": "1.0.19",
     "@bcrs-shared-components/base-address": "2.0.3",
     "@bcrs-shared-components/breadcrumb": "2.1.15",
-    "@bcrs-shared-components/business-lookup": "1.3.3",
+    "@bcrs-shared-components/business-lookup": "1.3.4",
```

Comment:
In this updated component, the default entity types peop now includes C, CBEN, CCC and CUL. (Some business searches use the default; other request specific types only.)